src/routes/raids/ViewRaid.tsx

```diff
@@ -19,7 +19,7 @@ const userStatSortNames = Object.keys(userStatSorts)
 
 const ViewRaid = () => {
   const { raidId } = useParams<{ raidId: string }>()
-  const documentData = useDocument<ViewRaidData>('raid-stats', raidId)
+  const documentData = useDocument<Raid>('raid-stats', raidId)
```

Comment:
Why was this changed?

Reply:
I'll admint `ViewRaidData` wasn't the best of names... I think `RaidData` would make more sense than `Raid` in this case, though

Reply:
Suffixing `Data` to stuff is superfluous, most things are data, it's the other way around that we want to tag it.
for example, when making a database model of a raid, would you call the entity `RaidData`? or `Raid`?

Reply:
I'd rather the suffix be there, maybe `RaidStats` instead. There will be more things in the future related to Raid and that will get convoluted fast.

Reply:
`RaidStats` sound good, doing that